bins/iotcored/bin/iotcored.c

```diff
@@ -57,6 +59,20 @@ static struct argp argp = { opts, arg_parser, 0, doc, 0, 0, 0 };
 int main(int argc, char **argv) {
     static IotcoredArgs args = { 0 };
 
+    // NOLINTBEGIN(concurrency-mt-unsafe)
```

Comment:
Could you leave a comment mentioning this is safe as long as getenv is re-entrant on the target and setenv is not called by other threads?

Reply:
Oops, I think I have two different spots where getenv is called... Should probably be in entry.c where the storage for the loaded config values is located.